src/types.md

```diff
@@ -508,10 +508,10 @@ Because captures are often by reference, the following general rules arise:
 
 > **<sup>Syntax</sup>**  
 > _TraitObjectType_ :  
-> &nbsp;&nbsp; _LifetimeOrPath_ ( `+` _LifetimeOrPath_ )<sup>\*</sup> `+`<sup>?</sup>
+> &nbsp;&nbsp; `dyn`<sup>?</sup> _LifetimeOrPath_ ( `+` _LifetimeOrPath_ )<sup>\*</sup> `+`<sup>?</sup>
 >
 > _LifetimeOrPath_ :
-> &nbsp;&nbsp; [_Path_] | [_LIFETIME_OR_LABEL_]
+> &nbsp;&nbsp; [_Path_] |  `(` [_Path_] `)` | [_LIFETIME_OR_LABEL_]
 
 A *trait object* is an opaque value of another type that implements a set of
```

Comment:
Do we want to update the language trait object -> dynamic trait everywhere? Otherwise looks good.

Reply:
I saw the internals discussion on this last night, but I didn't see any consensus on naming. Is there more I'm missing?